config/v1/types_ingress.go

```diff
@@ -32,8 +32,31 @@ type IngressSpec struct {
 	//
 	// Once set, changing domain is not currently supported.
 	Domain string `json:"domain"`
+
+	// scope is the default publishing scope for new ingress controllers on this
+	// cluster which are exposed by an OpenShift-managed load balancer.
+	//
+	// The default is "External".
+	//
+	// Changes to this field after creation will only be recognized by new
+	// ingress controllers.
+	// +optional
+	Scope *LoadBalancerScope `json:"scope,omitempty"`
```

Comment:
Is there any escaping using a pointer in this case?

Reply:
> Is there any escaping using a pointer in this case?

This would be no pointer, allow an empty string.